workers/unity/Packages/com.improbable.gdk.core/Tests/Editmode/Systems/SpatialOSReceiveSystemTests.cs

```diff
@@ -526,7 +530,10 @@ public void OnEntityQueryResponse_should_add_received_responses_to_entity()
                 Assert.AreEqual(emptyRequest, response.RequestPayload);
                 Assert.AreEqual(context, response.Context);
                 Assert.AreEqual(TestCommandRequestId, response.RequestId);
-                Assert.AreEqual(wrappedOp.Op, response.Op);
+                Assert.AreEqual(wrappedOp.Op.StatusCode, response.StatusCode);
+                Assert.AreEqual(wrappedOp.Op.Message, response.Message);
+                Assert.AreEqual(wrappedOp.Op.Result, response.Result);
+                Assert.AreEqual(wrappedOp.Op.ResultCount, response.ResultCount);
```

Comment:
@jamiebrynes7 is this an appropriate substitution for checking if wrappedOp matches response?

Reply:
discussed offline, `Assert.AreEqual` works for nulls (if both are null, they're equal)